File: Solutions/140035/python3/program.py
import math


def is_odd(number: int) -> bool:
    return number % 2 != 0


def is_prime(number: int) -> bool:
    for i in range(2, int(math.sqrt(number)) + 1):
        if (n % i) == 0:
            return False
    return True


n = int(input())

if is_odd(n) and is_prime(n):
    print("zoj")
else:
    print("fard")

File: Solutions/140035/csharp/Program.cs
using System;

namespace Quera {
    public static class Program {
        public static void Main() {
            var number = Convert.ToInt32(Console.ReadLine());

            if (IsOdd(number) && IsPrime(number)) {
                Console.WriteLine("zoj");
            }
            else {
                Console.WriteLine("fard");
            }
        }

        private static bool IsPrime(int number) {
            if (number < 2) return false;
            if (number < 4) return true;

            for (var i = 2; i < Math.Sqrt(number) + 1; i++) {
                if (number % i == 0)
                    return false;
            }

            return true;
        }

        private static bool IsOdd(int number) => number % 2 != 0;
    }
}